templates/webapi/main/group_builds_functionality_view.html.ep

```diff
@@ -40,7 +40,7 @@
                 </abbr>
             </div>
             <div class="px-2 align-self-end flex-grow-1">
-                %= include 'main/build_progressbar', max_jobs => $child_res->{total}, result => $child_res, class => "mt-0"
+                %= include 'main/build_progressbar', max_jobs => $child_res->{total}, params => undef, result => $child_res, class => "mt-0"
```

Comment:
And by the way, params is also not supposed to be `undef` here. Passing the actual parameters is still TODO.